WebHostLib/templates/weightedOptions/macros.html

```diff
@@ -19,7 +19,7 @@
             {% for id, name in option.name_lookup.items() %}
                 {% if name != 'random' %}
                     {% if option.default != 'random' %}
-                        {{ RangeRow(option_name, option, option.get_option_name(id), name, False, name if option.get_option_name(option.default)|lower == name else None) }}
+                        {{ RangeRow(option_name, option, option.get_option_name(id), name, False, name if option.get_option_name(option.default) == option.get_option_name(id) else None) }}
```

Comment:
Why not just `name if option.default == id else None`? It seems to work when I test it.

![image](https://github.com/ArchipelagoMW/Archipelago/assets/16137441/8e1eecec-b5d0-4eff-aedc-68a5eb86c69f)

IMO, that would be the best solution, and it shortens the line.

Reply:
Lol, sometimes I don't see the forest for the damn trees.